tests/tests_modifications_with_selinux_disabled.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: Ensure the default is targeted, enforcing, without local modifications
+  hosts: all
+  gather_facts: true
+  vars:
+    selinux_all_purge: true
+    selinux_logins_purge: true
+
+  tasks:
+    - name: Ensure SELinux tool semanage
+      package:
+        name:
+          - policycoreutils-python-utils
+        state: present
+      when: ansible_distribution == "Fedora" or
+        (ansible_distribution_major_version | int > 7 and
+         ansible_distribution in ["CentOS", "RedHat", "Rocky"])
+
+    - name: Ensure SELinux tool semanage on EL7
+      package:
+        name:
+          - policycoreutils-python
+        state: present
+      when:
+        - ansible_distribution_major_version | int == 7
+        - ansible_os_family == "RedHat"
+
+    - name: Ensure selinux-policy-targeted
+      package:
+        name:
+          - selinux-policy-targeted
+        state: present
+
+    - name: Add a Linux System Roles SELinux User
+      user:
+        comment: Linux System Roles SELinux User
+        name: sar-user
+    - name: Backup original /etc/selinux/config
+      copy:
+        remote_src: true
+        src: /etc/selinux/config
+        dest: /etc/selinux/config.test_selinux_disabled
+        mode: preserve
+    - name: Upload testing /etc/selinux/config
+      copy:
+        src: selinux.config
+        dest: /etc/selinux/config
+        mode: preserve
+    - name: Switch to permissive to allow login when selinuxfs is not mounted
+      command: setenforce 0
+      changed_when: false
+      when: ansible_selinux.status != "disabled"
+    - name: Get selinuxfs mountpoint
+      command: findmnt -n -t selinuxfs --output=target
+      changed_when: false
+      register: selinux_mountpoint
+    - name: >-
+        Umount selinux mountpoint to emulate SELinux disabled
+        system {{ selinux_mountpoint.stdout }}
+      command: umount -l {{ selinux_mountpoint.stdout }}
+      changed_when: false
```

Comment:
should this task have a `when` condition like `when: selinux_mountpoint.stdout != ""` e.g. don't execute the task if there is nothing to be done?  Then you can get rid of the `changed_when: false`

Reply:
`changed_when: true` is still necessary even with `when: selinux_mountpoint.stdout != ""`

https://github.com/linux-system-roles/selinux/actions/runs/6420308259/job/17432108183?pr=201

```

no-changed-when: Commands should not change things if nothing needs doing.
Error: Commands should not change things if nothing needs doing.
tests/tests_modifications_with_selinux_disabled.yml:29 Task/Handler: Umount selinux mountpoint to emulate SELinux disabled system {{ selinux_mountpoint.stdout }}

Read documentation for instructions on how to ignore specific rule violations.

                  Rule Violation Summary                  
 count tag             profile rule associated tags       
     1 no-changed-when shared  command-shell, idempotency 
```